butterfree/migrations/database_migration/cassandra_migration.py

```diff
@@ -1,16 +1,35 @@
 """Cassandra Migration entity."""
 
-from typing import Any, Dict, List
+import logging
+from typing import Any, List, Set
 
 from butterfree.clients import CassandraClient
 from butterfree.configs.db import CassandraConfig
 from butterfree.migrations.database_migration.database_migration import (
     DatabaseMigration,
+    Diff,
 )
 
 
 class CassandraMigration(DatabaseMigration):
-    """Cassandra class for Migrations."""
+    """Cassandra class for performing migrations.
+
+    This class implements some methods of the parent DatabaseMigration class and
+    has specific methods for query building.
+
+    The CassandraMigration class will be used, as the name suggests, for applying
+    changes to a given Cassandra table. There are, however, some remarks that need
+    to be highlighted:
+        - If an existing feature has its type changed, then it'll be dropped and a new
+        column with the same name and the new type will be created, therefore a
+        backfilling job may be required;
```

Comment:
I guess this is not true, since you seem to be using the `ALTER` command [here](https://github.com/quintoandar/butterfree/pull/291/files#diff-01efe2a031a6dffeeac227c42aa17668c6c2a9d507be98fcb1d90c9aaa131caeR51)

Reply:
Thanks, I've updated the docstrings from ```CassandraMigration``` class and its methods.